packages/flutter/lib/src/material/theme.dart

```diff
@@ -161,6 +161,42 @@ class Theme extends StatelessWidget {
         .resolveFrom(context);
   }
 
+  /// Retrieves the [Brightness] to use for descendant Material widgets, based
+  /// on the value of [ThemeData.brightness] in the given [context].
+  ///
+  /// If no [InheritedTheme] can be found in the given [context], or its `brightness`
+  /// is null, it will fall back to [MediaQueryData.platformBrightness].
```

Comment:
```suggestion
  /// is null, it will fall back to [MediaQueryData.platformBrightness], or null if there is no [MediaQuery] for the given [context].
```